index.html

```diff
@@ -1924,6 +2020,10 @@ <h4><code>ttp:contentProfiles</code></h4>
             <a>DAPT 1.0 Content Profile</a> designator.
```

Comment:
Why don't we just change this to "DAPT Documents" ?

Reply:
Done in f8c86e4 - found 3 instances.

Reply:
By mandating the 1.0 designator to be always present, we're making a strong promise that it will always be possible for a creator of a vNext document to make it backwards compatible and to want it backwards compatible. We may want to relax that and say that a DAPT document MUST have a DAPT designator and that this version currently defines only one.